components/scoreHandler.ts

```diff
@@ -848,7 +848,7 @@ export async function missionEnd(
             }
         }
 
-        calculateScoreResult.silentAssassin = false
+        contractScore.SilentAssassin = false
```

Comment:
There better fix here is to set both to false, not replace one with the other. Also counts for the .star line below.

The same is true for the sniper contract start, it's also only modifying the calculateScoreResult, not both.

The better fix would be to move the `contractScore` to beyond the contract-type checks so we can keep modifying the calculateScoreResult. But we would need a way to signal the output to include it or not (eg. how we now set contractScore to undefined).

Reply:
changed as requested